OutOfSchool/OutOfSchool.WebApi/Controllers/ChatWorkshopController.cs

```diff
@@ -56,30 +56,179 @@ public class ChatWorkshopController : ControllerBase
         }
 
         /// <summary>
-        /// Get a chat room with information about Parent and Workshop.
+        /// Get parent's chat room with information about Parent and Workshop.
         /// </summary>
         /// <param name="id">ChatRoom's Id.</param>
         /// <returns>User's chat room that was found.</returns>
-        [HttpGet("chatrooms/{id}")]
+        [HttpGet("parent/chatrooms/{id}")]
+        [Authorize(Roles = "parent")]
```

Comment:
nameof(Roles.parent) ?

Reply:
This is a great idea. Also, this issue is common for the entire solution, so first, we need to rename the enum to lower case. I think we can make it in a separate task.